### ISSUE TITLE
The canvas is flickering?

### ISSUE THREAD

**Issue by JokerHerry** [jloehr/AR-Sandbox#3]
I find is so flickering in running interface , can you have some good idea?

**Comment by JokerHerry** [comment]
hello ,can you hear me , i think i already solve the problem now,use Weighted Moving Average,[https://www.codeproject.com/Articles/317974/KinectDepthSmoothing](url),but i have a new problem,how to add the contour line,can you give me some lighting,    (๑•̀ㅂ•́)و✧
Thanks !

**Comment by jloehr** [comment]
Yes, the flickering comes from the high noise of the Kinect Sensor and the fact that the raw frame data is used. Any kind of smoothing or filtering will help.
